src/config/mocks/adapters/review.js

```diff
@@ -0,0 +1,34 @@
+import { createPaginationResponse, extractIdPathParamFromUrl } from '../common/helpers';
+import review from '../data/review';
```

Comment:
the data file should be plural reviews
```suggestion
import reviews from '../data/reviews';
```